src/egon/data/datasets/electrical_neighbours.py

```diff
@@ -1536,6 +1520,14 @@ def insert_generators_sq(gen_sq=None, scn_name="status2019"):
     # Add marginal costs
     list_gen_sq = add_marginal_costs(list_gen_sq)
 
+    # assign marginal cost for OCGTs
+    scn_params_gas = get_sector_parameters("gas", scn_name)
+    efficiency_ocgt = scn_params_gas["efficiency"]["OCGT"]
+    list_gen_sq.loc[list_gen_sq.carrier=="OCGT","marginal_cost"] = (
+        scn_params_gas["marginal_cost"]["OCGT"]
+        / efficiency_ocgt
+    )
```

Comment:
These costs do not include fuel costs for the gas

Reply:
Corrected in 59db364